flow/Makefile

```diff
@@ -33,7 +33,7 @@
 # DESIGN_CONFIG=./designs/tsmc65lp/vanilla5/config.mk
 
 # DESIGN_CONFIG=./designs/gf12/aes/config.mk
-# DESIGN_CONFIG=./designs/gf12/ariane/config.mk
```

Comment:
Don't change default design

Reply:
Yes, I saw that -- committed by mistake. The fix is syncing from the private repo